core/service.py

```diff
@@ -134,7 +134,7 @@ def setup_restful_service(app):
     spec.path(resource=CommentResource, api=api)
     api.add_resource(TagResource, '/<any(file, config, blob, object):type>/<hash64:identifier>/tag')
     spec.path(resource=TagResource, api=api)
-    api.add_resource(ShareResource, '/share', '/<any(file, config, blob, object):type>/<hash64:identifier>/share')
+    api.add_resource(ShareResource, '/<any(file, config, blob, object):type>/<hash64:identifier>/share')
```

Comment:
what is the effect of this change?

Reply:
`/share` is no longer accepted (variant without arguments), described already in PR description